### ISSUE TITLE
[BUG] version 2.2.2 - error while executing main procedure

### ISSUE THREAD

**Issue by caissist** [limosoft-de/raffytimelapse#20]
[RaffyTimelapse - error while executing main procedure -2020.06.17.txt](https://github.com/limosoft-de/raffytimelapse/files/4809828/RaffyTimelapse.-.error.while.executing.main.procedure.-2020.06.17.txt)
**Describe the bug**
Error message popped up about at the end of main procedure. I was able to execute procedure to the end though.

**Screenshots**
See enclosed error message file.

**Symstem Information**
 - OS: Windows 7 Prof 64-bit
 - Raffy Timelapse Version: 2.2.2

**Comment by limosoft-de** [comment]
This looks to me like the temp files couldn't be deleted, as the folder wasn't empty. Unsure why this did happen, maybe due to missing rights on your computer for that user? Never had anything like that reported yet. Do you remember if you exported your footage as .flv by any chance?

I will have to double-check the code to see why exactly the program is expecting an empty folder by default at this point, as it is stated in the log. If there isn't any valid reason for it, passing a `recursive ` parameter as `true` should solve this problem (see [MSDN](https://docs.microsoft.com/de-de/dotnet/api/system.io.directoryinfo.delete?view=netcore-3.1#System_IO_DirectoryInfo_Delete_System_Boolean_)).

I will check this out and implement it in the next release if it is feasible. Thank you very much for reporting this.

**Comment by limosoft-de** [comment]
I just checked the code, and found that the `FileSystem.DeleteDirectory` Method is being used, and that the `DeleteAllContents `option is being properly passed as well (see [MSDN](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualbasic.fileio.filesystem.deletedirectory?view=netcore-3.1)). Therefor, the folder should be recursively removed, whether it contains any files or not. I honestly have no clue why this isn't happening, as your error log clearly states, that the folder wasn't empty, which lead to the exception. 

As the method isn't that well documented I did replace it with `System.IO.Directory.Delete` (see [MSDN](https://docs.microsoft.com/de-de/dotnet/api/system.io.directoryinfo.delete?view=netcore-3.1)) method in https://github.com/limosoft-de/raffytimelapse/commit/511fae3ad2c1f94e87d4b2dafb8294f39deae583, with the `recursive` parameter set to `true`. Hoping this will be more robust and help solve this issue.

Will be released with the next update.
